documentation/source/newsfragments/1425.change.rst

```diff
@@ -0,0 +1,3 @@
+All libraries are compiled during installation to the ``cocotb/libs`` directory.
+The interface libraries ``libcocotbvpi`` and ``libcocotbvhpi`` have been renamed to have a ``_simulator_name`` postfix.
+The ``simulator`` module has moved to :mod:`cocotb.simulator`.
```

Comment:
```suggestion
The ``simulator`` module has moved to :mod:`cocotb.simulator`.
The ``LD_LIBRARY_PATH`` environment variable is no longer modified by cocotb, as the libraries now discover each other via ``RPATH`` settings.
```

Reply:
I can imagine people being broken somehow, and having `LD_LIBRARY_PATH` appear somewhere in the release notes will likely be the red flag they're looking for.